src/main/scala/io/iohk/ethereum/crypto/SignatureValidator.scala

```diff
@@ -0,0 +1,57 @@
+package io.iohk.ethereum.crypto
+
+import akka.util.ByteString
+import io.iohk.ethereum.crypto
+import io.iohk.ethereum.jsonrpc.JsonMethodsImplicits
+import io.iohk.ethereum.nodebuilder.SecureRandomBuilder
+import io.iohk.ethereum.utils.ByteStringUtils
+
+import scala.util.{Failure, Success, Try}
+
+// scalastyle:off regex
+object SignatureValidator extends App with SecureRandomBuilder with JsonMethodsImplicits {
+
+  args match {
+    case Array(pk, sig, msgHash) =>
+      Try {
+        val signature = ECDSASignature.fromBytes(ByteString(decode(sig)))
```

Comment:
We should probably move this decoder to ByteStringUtils and not rely here on the jsonrpc methods